lib/src/git.rs

```diff
@@ -233,49 +256,43 @@ pub fn export_refs(
     })
 }
 
-/// Short-term TODO: Docstring is in the next commit
+/// Exports unconflicted branches.
+///
+/// Potential conflicts between the Git repo state and the JJ state are detected
+/// by using `old_view` as the conflict base. Returns the list of unexported
+/// branches and the new git state (with successfully exported branches moved).
 fn export_branches_since_old_view(
     mut_repo: &mut MutableRepo,
-    // Short-term TODO: Currently, this function ignores old view and always returns an empty
-    // GitRefView
-    _old_view: &GitRefView,
+    old_view: &GitRefView,
     git_repo: &git2::Repository,
 ) -> Result<(GitRefView, Vec<String>), GitExportError> {
-    // First find the changes we want need to make without modifying mut_repo
+    let current_view = mut_repo.view();
+    let current_branches: BTreeSet<_> = current_view.branches().keys().cloned().collect();
+    let old_branch_names: BTreeSet<_> = old_view
+        .refs
+        .keys()
+        .filter_map(|r| ref_name_to_local_branch_name(r))
+        .collect();
```

Comment:
My first attempt at this didn't work since it resulted with an immutable reference to `mut_repo` that couldn't exist because a mutable reference is also needed.

Feel free to suggest a fix, but I'm dropping this for now.

Reply:
No problem, sounds good to drop it.